### ISSUE TITLE
Color Contrast issues

### ISSUE THREAD

**Issue by clapierre** [onikamigames/starshipgame#111]
There are multiple places where the color contrast is very difficult to read text.
Especially for example the amount of time returning from a planet is
![green text on blue background](https://user-images.githubusercontent.com/7736749/145920523-62ebc52d-ef39-4b3e-9d2c-121bd8cc3511.png)

This has a 1.2 : 1 contrast ratio.
WCAG-AA compliance should have at least a 4.5:1 contrast ratio or 3:1 for large text.
Here is a free tool to check the color contrast ratios

[Colour Contrast Analyser](https://www.tpgi.com/color-contrast-checker/)	free tool from TPGi to check the color contrast conformance of foreground and background colors of text
